homeassistant/components/openweathermap/__init__.py

```diff
@@ -62,7 +62,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return True
 
 
-async def async_migrate_entry(hass, entry):
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
Please type one module completely per PR instead. That'll make the scope appropriate and will let us keep track of what parts of an integration is done with adding type annotations.